### PR TITLE
Fixes #1 Homepage now has philareads text and link to contributing

### LINKED ISSUE #1: Home page doesn't have any philareads content

### PR DESCRIPTION
**Pull request by Davidw1339**
https://phillyreads-fgwyieilem.now.sh/

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -8,16 +8,14 @@ class App extends Component {
       <div className="App">
         <header className="App-header">
           <img src={logo} className="App-logo" alt="logo" />
```
Comment (XuanLi-Leong) [inline frontend/src/App.js]: this is naaaaaht the philareads logo 😿

File: frontend/src/App.js [hunk 2]
```diff
@@ -8,16 +8,14 @@ class App extends Component {
...
-          <p>
-            Edit <code>src/App.js</code> and save to reload.
-          </p>
+          <p>Welcome to PhilaREADS homepage!</p>
```
Comment (Davidw1339) [inline frontend/src/App.js]: good job :+1:

File: frontend/src/App.js (hunk 1 above)
Comment (alicesf2) [reply frontend/src/App.js]: tru

File: frontend/src/App.js [hunk 3]
```diff
@@ -8,16 +8,14 @@ class App extends Component {
...
           <a
             className="App-link"
-            href="https://reactjs.org"
+            href="https://github.com/hack4impact-uiuc/philareads/blob/master/contributing.md"
```
Comment (micleo2) [inline frontend/src/App.js]: :+1:  helpful link

Comment (XuanLi-Leong) [reply frontend/src/App.js]: 💯

File: frontend/src/App.js [hunk 4]
```diff
@@ -8,16 +8,14 @@ class App extends Component {
...
             target="_blank"
             rel="noopener noreferrer"
           >
-            Learn React
+            Contribute!
```
Comment (melodyfelix) [inline frontend/src/App.js]: but i would also like to learn react 😢

**Pull request closed without merging**